engine/src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -157,7 +157,11 @@ trait SharedFileSystem { self: Backend =>
   /**
    * Creates host execution directory.
    */
-  def initializeForWorkflow(descriptor: WorkflowDescriptor): Try[Unit] = Try {
+  def initializeForWorkflow(descriptor: WorkflowDescriptor): Future[Unit] = {
+    Future.fromTry(makeExecutionDirectory(descriptor))
```

Comment:
Can you make this async by passing in an `ExecutionContext` instead of blocking during a `Try`?